cli/pcluster/config/mappings.py

```diff
@@ -796,35 +911,41 @@
                 )
             }),
             ("efs_settings", {
-                "type": SettingsParam,
+                "type": SettingsCfnParam,
                 "referred_section": EFS,
                 "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
             ("raid_settings", {
-                "type": SettingsParam,
+                "type": SettingsCfnParam,
                 "referred_section": RAID,
                 "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
             ("fsx_settings", {
-                "type": SettingsParam,
+                "type": SettingsCfnParam,
                 "referred_section": FSX,
                 "validators": [fsx_os_support],
                 "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
             ("dcv_settings", {
-                "type": SettingsParam,
+                "type": SettingsCfnParam,
                 "referred_section": DCV,
                 "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
             ("cw_log_settings", {
-                "type": SettingsParam,
+                "type": SettingsCfnParam,
                 "referred_section": CW_LOG,
                 "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
+            ("queue_settings", {
```

Comment:
Yes, sorry this was still opened because of these two pending questions:

1: do we want to apply the same logic to all settings params? I'd vote for yes
2: should we also check for a max length? I'd vote for yes and with a max length of 64 chars

Reply:
I'm onboard with both

Reply:
Done